src/components/pathway/SearchPathwayCard.jsx

```diff
@@ -85,7 +75,7 @@ const SearchPathwayCard = ({ hit, isLoading }) => {
           );
         }}
       >
-        <Card>
+        <Card className={classNames({ 'skill-quiz-pathway-card': isSkillQuizResult })}>
```

Comment:
`isSkillQuizResult` is it a class name?

Reply:
It's a flag we pass when we call PathwayCard component from skill quiz.